plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageViewModel.java

```diff
@@ -407,11 +412,11 @@ public boolean hasSourceCode() {
     }
 
     /**
-     * Checks whether change coverage exists.
+     * Checks whether Modified Lines Coverage exists.
      *
-     * @return {@code true} whether change coverage exists, else {@code false}
+     * @return {@code true} whether Modified Lines Coverage exists, else {@code false}
      */
-    public boolean hasChangeCoverage() {
+    public boolean hasModifiedLinesCoverage() {
```

Comment:
These changes prevent that the table is shown in the UI. I think a change to the `index.jelly` is missing.